drivers/adc/Kconfig

```diff
@@ -303,4 +303,12 @@ endif
 
 endif # ADC_QMSI || ADC_QMSI_SS
 
+config ADC_MCUX_ADC16
+	bool "MCUX ADC16 driver"
+	depends on HAS_MCUX && HAS_ADC16
+	select HAS_DTS_ADC
```

Comment:
It doesn't look it in GitHub, but I actually added the dts first in the patch series before the driver. GitHub appears to show patches ordered by AuthorDate rather than CommitDate, so the fact that I originally implemented this driver without dts, amended it, and inserted a new commit adding dts before it, got GitHub a little confused.